src/medCore/gui/commonWidgets/medSlider.h

```diff
@@ -28,6 +28,7 @@ public slots:
 
 protected:
     void paintEvent(QPaintEvent *ev);
+    void mousePressEvent ( QMouseEvent * event );
```

Comment:
Could you please just remove the spaces so this is stylised the same way as above ?